extractor/filesystem/list/list.go

```diff
@@ -103,6 +104,8 @@ var (
 	Dart []filesystem.Extractor = []filesystem.Extractor{pubspec.Extractor{}}
 	// Erlang extractors.
 	Erlang []filesystem.Extractor = []filesystem.Extractor{mixlock.Extractor{}}
+	// Haskell extractors.
+	Haskell []filesystem.Extractor = []filesystem.Extractor{cabal.Extractor{}}
```

Comment:
We have a constructor function for this extractor, so let's useNew(DefaultConfig()) here.